src/common/store/customBurgerStore/customBurgerReducer.ts

```diff
@@ -35,15 +75,149 @@ const initialState: BurgerState = {
       choose: false,
     },
   ],
+  totalPrice: 2.0,
 };
 
 const customBurgerReducer = (state: BurgerState = initialState, action: CustomBurgerActionTypes): BurgerState => {
+  let ingPrice = 0;
```

Comment:
I would suggest you moving this variable to the specific cases

Reply:
I will move it out of the reducer function and treat it as a global variable.